web/admin/application/configs/klear/conf.d/mapperList.yaml

```diff
@@ -15,6 +15,10 @@ mappers:
         modelFile: BalanceNotifications,
         entity: 'Ivoz\\Provider\\Domain\\Model\\BalanceNotification\\BalanceNotification'
     }
+  - &BillableCalls {
+        modelFile: BillableCalls,
+        entity: 'Ivoz\\Provider\\Domain\\Model\\BillableCalls\\BillableCalls'
```

Comment:
entity in singular _'Ivoz\\Provider\\Domain\\Model\\BillableCall\\BillableCall'_